api/src/test/java/org/pmiops/workbench/api/CohortReviewControllerTest.java

```diff
@@ -169,7 +177,13 @@ public long getConceptId() {
   private static DbUser user;
 
   @TestConfiguration
-  @Import({CdrVersionService.class, CohortReviewController.class})
+  @Import({
+    CdrVersionService.class,
```

Comment:
do you have a macro for alphabetizing lines like this?

Reply:
I don't.